app/lib/services/ecf/ecf_user_creator.rb

```diff
@@ -32,6 +32,9 @@ def call
         Sentry.with_scope do |scope|
           scope.set_context("User", { id: user.id })
           Sentry.capture_exception(e)
+
+          # Re-raise to fail the sync, we'll want to retry again later
+          raise e
```

Comment:
This is how the class used to function before I added in the exception capturing.
 
Previously if a sync failed in a way that raised an exception (which includes failed saves since JsonApiClient::Resource uses errors for flow control) then the ApplicationSubmissionJob would fail and be retried later, which may allow the error to resolve itself. Without it the job would succeed and no retry would occur.